src/header.py

```diff
@@ -20,6 +20,7 @@
 import sys
 from typing import Callable, Dict
 import functools
+from packaging import version
```

Comment:
Unfortunately, versioneer needs to be pure stdlib to be vendorable. `packaging` is common, but can't be guaranteed in a build environment.

Reply:
Hmm.. Ok. What if I leave only pre-1.7.2 code with `--left-right` without any choosing of version? Will it be OK?

Reply:
Yes, that seems fine.

Reply:
Done.